docs/user-event/api-utility.mdx

```diff
@@ -0,0 +1,179 @@
+---
+id: utility
+title: Utility APIs
+---
+
+The following APIs don't have one-to-one equivalents in a real user
+interaction.  
+Their behavior is therefore an interpretation how the "perceived" user
+interaction might be translated to actual events on the DOM.
+
+## clear()
+
+```ts
+clear(element: Element): Promise<void>
+```
+
+This API can be used to easily clear an editable element.
+
+1. Focus element
```

Comment:
```suggestion
The events we initiate are:
1. Focus element
```

Reply:
I'm not sure about this. `events` would be misleading I think. This should describe the steps the API performs as the steps performed by these APIs have not been clear to our users in the past.

Reply:
So maybe we should write:
```suggestion
This API mimics a "clear" action by running these steps:
1. Focus element
```